modules/javafx.graphics/src/main/java/com/sun/javafx/font/coretext/CTGlyph.java

```diff
@@ -86,8 +86,6 @@ private void checkBounds() {
         xAdvance = size.width;
         yAdvance = -size.height;   /*Inverted coordinates system */
 
-        if (drawShapes) return;
-
```

Comment:
Just curious: do you know why this test was there in the first place? And why it needs to be removed?

Reply:
My guess is that the info it gathered was not used in the glyph image path
and it was an attempt at optimisation.
As to why it needed to be removed, it is because  I DO need that information and there'll be a NPE (IIRC) if I don't remove it.